src/D3MHub.sol

```diff
@@ -560,15 +576,6 @@ contract D3MHub {
         require(art <= MAXINT256, "D3MHub/overflow");
         vat.grab(ilk, address(_pool), address(_pool), vow, -int256(ink), -int256(art));
 
-        if (ink > art) {
```

Comment:
just a reminder to update the comments in the `cull` NatSpec and consider documenting `fix` behaviour as well in `exec`

Reply:
Comment fixed!